alot/commands/envelope.py

```diff
@@ -205,6 +222,18 @@ def __init__(self, mail=None, envelope=None, **kwargs):
                                     msg_position='left')) == 'no':
                     return
 
+            # Check if an encrypted message is indeed encrypted to all its
+            # recipients.
+            if self.envelope.encrypt \
+                    and not self._is_encrypted_to_all_recipients():
```

Comment:
I'd like to see parens instead of \ here:
```python
if (self.envelope.encrypt
        and not self._is_encrypted_to_all_recipients()):
```

This is in keeping with the style of the ```if (await...``` below